third_party/basisu/tnt/CMakeLists.txt

```diff
@@ -24,19 +24,33 @@ set(ENCODER_SRC
     ../zstd/zstd.c
 )
 
+# Filament does not support certain compressed formats that BasisU can transcode
+# to, so we disable them in the build to save space. Some of these mappings are
+# confusing. (e.g., DXT5A corresponds to transcoder_texture_format::cTFBC4_R).
+set (BASIS_CONFIG
+    BASISD_SUPPORT_KTX2=1
+    BASISD_SUPPORT_KTX2_ZSTD=1
+    BASISD_SUPPORT_ATC=0
+    BASISD_SUPPORT_DXT5A=0
+    BASISD_SUPPORT_BC7=0
```

Comment:
Isn't BC7 super useful on desktop?

Reply:
Filament does not support BC7 yet.  Here's a quote from the BasisU wiki:

> If you know your platform doesn't support BC7 or it's not necessary, be sure to set BASISD_SUPPORT_BC7=0. The tables for the mode 6 transcoder are very large (they bloat the compiled WebAssembly transcoder by around 400-500k). We'll probably be removing the mode 6 transcoder in a future update.